hack/local-running.sh

```diff
@@ -36,7 +36,7 @@ function create_cluster() {
   kind create cluster --name "${cluster_name}" --kubeconfig="${kubeconfig}" --image="${image}" 2>&1
 
   kubectl config rename-context "kind-${cluster_name}" "${cluster_name}" --kubeconfig="${kubeconfig}"
-  kind_server="https://$(docker inspect --format='{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' "${cluster_name}-control-plane"):6443"
+  kind_server="https://$(docker inspect --format='{{(index (index .NetworkSettings.Ports "6443/tcp") 0).HostIp}}:{{(index (index .NetworkSettings.Ports "6443/tcp") 0).HostPort}}' "${cluster_name}-control-plane")"
```

Comment:
IIRC, current script works well. Have you got any exceptions/tracing logs?

Reply:
Hi, I've been stuck at helm installing.
```
+ echo 'Installing clusternet-hub...'
Installing clusternet-hub...
+ helm --kubeconfig=/Users/bart/.kube/clusternet.config --kube-context=parent install clusternet-hub -n clusternet-system --create-namespace clusternet/clusternet-hub
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /Users/bart/.kube/clusternet.config
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /Users/bart/.kube/clusternet.config
Error: INSTALLATION FAILED: Kubernetes cluster unreachable: Get "https://172.18.0.5:6443/version": read tcp 192.168.255.10:62049->172.18.0.5:6443: read: connection reset by peer
```